quodlibet/config.py

```diff
@@ -49,6 +49,7 @@
 
         "is_playing": "false",
         "restore_playing": "false",
+        "consider_played_percent": "50",
```

Comment:
Not sure about this name (_consider_ feels a bit weird).
```suggestion
        "playcount_minimum_length_proportion": "0.5",
```
or something along those lines?

Also, all (new) config items should have a comment to explain them

Reply:
Done.